src/main/scala/io/pubcrawler/stout/controllers/StopController.scala

```diff
@@ -0,0 +1,79 @@
+package io.pubcrawler.stout.controllers
+
+import math._
+import akka.util.Timeout
+import io.pubcrawler.stout.db.{Stop, TableDefinitions}
+import io.pubcrawler.stout.models.{Participant, Result}
+import io.pubcrawler.stout.util.{JsonFormat, UtilAsync}
+import org.scalatra.json.JacksonJsonSupport
+import org.scalatra.swagger.{Swagger, SwaggerSupport}
+import org.scalatra.{FutureSupport, ScalatraServlet}
+import slick.jdbc.JdbcBackend.Database
+import slick.jdbc.PostgresProfile.api._
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+
+class StopController(implicit val db: Database, implicit val swagger: Swagger) extends ScalatraServlet
+  with JacksonJsonSupport with FutureSupport with TableDefinitions with JsonFormat with UtilAsync with SwaggerSupport {
+
+  protected implicit val timeout = new Timeout(2.seconds)
+
+  override protected def applicationDescription: String = "Stout Stop API"
+
+  private val R: Double = 6378.137 // earth radius in km
+  private val b: Double = 0.00892857142 // absolute maximum distance in lat or lng pr km
```

Comment:
This is a quick and dirty way to filter values in the database-query before performing haversine-distance between two points.

This value actually varies depending on where you are in the world; this value is a tad higher than maximum value.